### ISSUE TITLE
Ptx2 - anyone know if all is well with him??

### ISSUE THREAD

**Issue by nealjane** [ptx2/gymnasticon#91]
@ptx2 - noticed he’s not been on here for quite a while now, he used to respond to issues fairly quickly, which is what’s got me wondering about him - so I’m hoping all is OK with him and just taking a loooong break from it!

🚴🎄merry Christmas & a Happy new year to all my fellow Gymnasticon users!🎄🚴

**Comment by djfjeff** [comment]
Yeah, I hope everything is alright with him. I sent him an email about this project and never get anything back so I hope it's nothing bad.

**Comment by runwilder** [comment]
I know nothing about Raspberry Pi but have some novice programming skills... Bought a Flywheel bike from an auction site that appears brand new minus the computer. Hoping to follow @ptx2 guide to get it running with Zwift so I can bike with my friend who lives in another state... wish me luck... If it works out and @ptx2 is still around, I'd be happy to send a donation.

**Comment by nealjane** [comment]
It’s simple enough - just use the image and write to sd card. & Don’t forget to calibrate the bike!. 👍🏻
